geomstats/_backend/numpy/__init__.py

```diff
@@ -406,3 +406,14 @@ def mat_from_diag_triu_tril(diag, tri_upp, tri_low):
     mat[..., j, k] = tri_upp
     mat[..., k, j] = tri_low
     return mat
+
+def vec_to_sym(vec, mat_dim):
+    shape = (mat_dim, mat_dim)  
+    shape = (vec.shape[0],) + shape if vec.ndim>1 else shape
+    mat = np.zeros(shape)
+    i, = np.diag_indices(mat_dim, ndim=1)
+    j, k = np.tril_indices(mat_dim)
+    mat[... , j, k] = vec
```

Comment:
Is this differentiable?

Reply:
This, I am not sure. I have never used old `autograd`   :) . I will modify this probably. But pytorch one works